core/prisma/seed.ts

```diff
@@ -30,12 +30,10 @@ async function createUserMembers({
 	role: "editor" | "admin" | "contributor";
 	prismaCommunityIds: string[];
 }) {
-	let user;
-
 	await prisma.user.create({
 		data: {
 			slug,
-			email: user ? user.email : email,
```

Comment:
i wonder what i was thinking 🤔

Reply:
it did something at first, there was logic there to fetch the user from supabase, and if so, use the supabase users email. I just forgot to remove this in the cleanup!